platform/osx/bitrise.yml

```diff
@@ -33,6 +33,8 @@ workflows:
             export BUILDTYPE=Debug
             make osx
             make test-osx
+            make linux
+            make test-linux
```

Comment:
How much time does this add to the build? (We’re trying to keep build times down.) With these invocations, do we end up rebuilding everything from scratch?